src/api/app/models/owner_assignee_search.rb

```diff
@@ -0,0 +1,132 @@
+class OwnerAssigneeSearch < OwnerSearch
+  def for(search_string)
+    @match_all = limit.zero?
+    @deepest = (limit < 0)
+
+    @instances_without_definition = []
+    @maintainers = []
+
+    # search in each marked project
+    object_projects(search_string).each do |project|
+      @rolefilter = filter(project)
+      @already_checked = {}
+      @rootproject = project
+      @lookup_limit = limit.to_i
+      @devel_disabled = devel_disabled?(project)
+      find_assignees search_string
+      webui_mode = params[:webui_mode].present?
+      return @instances_without_definition if webui_mode && @maintainers.empty?
+    end
+    @maintainers
+  end
+
+  protected
+
+  def create_owner(pkg)
+    Owner.new(rootproject: @rootproject.name, filter: @rolefilter, project: pkg.project.name, package: pkg.name)
+  end
+
+  def extract_maintainer_project_level(owner, pkg)
+    return owner if owner.user_or_group?
+
+    owner.package = nil
+    extract_from_container(owner, pkg.project, @rolefilter)
```

Comment:
`extract_from_container` would return an owner. So it seems that this isn't doing anything. Or am I missing something (still trying to wrap my head around all this)?

Reply:
It fills the owner object